project/core/templates/support_us.html

```diff
@@ -123,13 +123,15 @@ <h4 class="center">{% trans "Spread the Word" %} </h4>
     </div>
 
 </script>
+{% endblock backbone_template %}
 
-<body>
+{% block content %}
     {% include "static_nav.html" %}
     <div id="supportus"></div>
     {% include "static_footer.html" %}
```

Comment:
Let's move the static navigation and footer to the `base.html`.
```suggestion
    <div id="supportus"></div>
```